examples/bri-3/src/bri/transactions/transactions.profile.ts

```diff
@@ -1,11 +1,5 @@
 import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
-import {
-  createMap,
-  forMember,
-  ignore,
-  mapFrom,
-  Mapper,
-} from '@automapper/core';
+import { createMap, forMember, mapFrom, Mapper } from '@automapper/core';
```

Comment:
2 more warnings in this file, build fails for errors only though, but github shows them for some reason below